configure.ac

```diff
@@ -32,7 +32,7 @@ AS_IF([test "x$RTE_SDK" != "x"], [
     AC_MSG_RESULT([$RTE_SDK])
     AC_SUBST(RTE_SDK_CFLAGS, ["-I$RTE_SDK/build/include -D RTE_MAX_LCORE=128 -D RTE_PKTMBUF_HEADROOM=128"])
     AC_SUBST(RTE_SDK_HEADERS_FOLDER, [$RTE_SDK/build/include/])
-    AC_SUBST(RTE_SDK_LIBS, [" -lm -lpthread -ldl -lpcap -L$RTE_SDK/build/lib/ -ldpdk "])
+    AC_SUBST(RTE_SDK_LIBS, [" -lm -lpthread -ldl -lpcap -lnuma -L$RTE_SDK/build/lib/ -ldpdk "])
```

Comment:
- typo in commit message, maybe a message like "update to dpdk v17.08" ?
- need to instal numa lib in travis
- update DPDK version in README.md